tests/Microsoft.Identity.Web.Test/AzureADB2COpenIDConnectEventHandlersTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Web.Test.Common.TestHelpers;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class AzureADB2COpenIDConnectEventHandlersTests
+    {
+        private const string PathBase = "/PathBase";
+        private const string DefaultUserFlow = "DefaultUserFlow";
+        private const string NonDefaultUserFlow = "NonDefaultUserFlow";
```

Comment:
maybe call this "CustomUserFlow", which would be the non-default, and the value similar to something above, at least which starts w/"b2c_1_*"

Reply:
Updated.